src/button-group/internal.tsx

```diff
@@ -107,10 +111,7 @@ const InternalButtonGroup = forwardRef(
     }
 
     function focusElement(element: HTMLElement) {
-      setFeedbackItemId(null);
-      setTimeout(() => {
```

Comment:
Removed setTimeout as all my automated and manual tests pass w/o it. If there is a scenario that requires a timeout let's cover it with a unit or integ test